### DISCUSSION TITLE
Make folders/ collection names searchable

### DISCUSSION THREAD

**Discussion by kompetenzlandkarte** [dani-garcia/vaultwarden#1923] [Ideas]
Hello together,

we noticed that we can't search for collection or folder names. We recently switched our org. from KeePass to Vaultwarden with great success and the users adapted pretty quickly. But it was pretty common in our environment that we searched for a server hostname and got the folder with it's belonging passwords.

This would be a great addition to vaultwarden!

Are there maybe some workarounds currently in place to achieve this already?

Thank you.

**Comment by BlackDex** [comment]
This is not a Vaultwarden specific feature. We use the web-vault from Bitwarden Upstream.
See: https://bitwarden.com/help/article/searching-vault/ for the searching features there currently are.
If you want to have this features added i suggest to start a feature request there.
